shared/control/parallel_dd.control

```diff
@@ -17,8 +17,8 @@ path.sort()
 fs = path[-1]
 platform = Popen(["uname", "-m"], stdout=PIPE).communicate()[0]
 if "ppc" in platform:
-    if not Popen("lsscsi", stdout=PIPE).communicate()[0]:
-        fs = path[0]
+    path = glob.glob("/dev/disk/by-id/*TARGET_DISK*")
+    fs = ('/dev/' + Popen(["ls", path[0], "-ls"], stdout=PIPE).stdout.readline().split('/')[-1]).split('\n')[0]
```

Comment:
If the behavior of x86 and ppc is identical.  How about remove 'if "ppc" in platform' ?  And let the x86 and ppc go with the same logic ?